tools/src/format.rs

```diff
@@ -475,7 +488,41 @@ pub fn format_path(cfg: &config::Config,
     let commit_obj = try!(git.repo.revparse_single(rev).map_err(|_| "Bad revision"));
     let commit = try!(commit_obj.into_commit().map_err(|_| "Bad revision"));
     let commit_tree = try!(commit.tree().map_err(|_| "Bad revision"));
-    let entry = try!(commit_tree.get_path(Path::new(path)).map_err(|_| "File not found"));
+    let path_obj = Path::new(path);
+    let data = match commit_tree.get_path(path_obj) {
+        Ok(entry) => try!(entry_to_blob(&git.repo, &entry)),
```

Comment:
You can use `?` if you want now, though we can probably migrate it altogether if you want :)